### PR TITLE
Added solutions for question 3 in lesson 4

### PR DESCRIPTION
**Pull request by AKuederle**
Added the missing solutions for the question 'Committing Multiple Files'.

### CODE REVIEW

File: _episodes/04-changes.md [hunk 1]
```diff
@@ -545,6 +545,54 @@ repository (`git commit`):
 > about Venus as a base for you and your friends
 > 3. Add changes from both files to the staging area,
 > and commit those changes.
+> > ## Solution
+> >
+> > First we make our changes to the 'mars.txt' and 'venus.txt' files:
```
Comment (rgaiacs) [inline _episodes/04-changes.md]: Change

- 'mars.txt' to `mars.txt`,
- 'venus.txt' to `venus.txt`

File: _episodes/04-changes.md [hunk 2]
```diff
@@ -545,6 +545,54 @@ repository (`git commit`):
...
+> > $ git add mars.txt venus.txt
+> > ~~~
+> > {: .bash}
+> > Or with multiple commands:
+> > ~~~
+> > $ git add mars.txt
+> > $ git add venus.txt
+> > ~~~
+> > {: .bash}
+> > Now the files are ready to commit. You can check that using `git status` first, or directly commit:
```
Comment (rgaiacs) [inline _episodes/04-changes.md]: What about,

> Now the files are ready to be committed and you can check using `git status`. To commit:

File: _episodes/04-changes.md [hunk 3]
```diff
@@ -545,6 +545,54 @@ repository (`git commit`):
...
+> > ~~~
+> > $ git commit -m"Wrote down my plans to start a base on Venus"
```
Comment (rgaiacs) [inline _episodes/04-changes.md]: Add one space after `-m`.

File: _episodes/04-changes.md [hunk 4]
```diff
@@ -545,6 +545,54 @@ repository (`git commit`):
...
+> > ~~~
+> > {: .bash}
+> > ~~~
+> > [master cc127c2]
+> > Wrote down my plans to start a base on venus
+> > 2 files changed, 2 insertions(+)
+> > ~~~
+> > {: .output}
+> > ~~~
+
```
Comment (rgaiacs) [inline _episodes/04-changes.md]: Please remove the empty lines here.

**Comment by iglpdc** [comment]
Thanks @AKuederle!

**Pull request merged**